src/client/node_transformations.js

```diff
@@ -42,7 +42,7 @@ const getXAndY = (angle, distance, startingCx, startingCy) => {
 
 const getMemoryNodePositions = (tagNode, numTagMemories, memoryIndex, currentLikes) => {
   const angle = ((2 * Math.PI) / numTagMemories) * memoryIndex;
-  const distance = (tagNode.likes - currentLikes) * 2;
+  const distance = (tagNode.likes - currentLikes) * 2 || 50;
```

Comment:
That `||` is weird

Work out explicity what you are trying to catch then use a ternary.

Something like
```js
tagNode.likes !== undefined ? (tagNode.likes - currentLikes) * 2 : 50
```